test/integration/helpers_test.go

```diff
@@ -183,3 +187,32 @@ func gatewayLinkStatusMatches(t *testing.T, c *gatewayclient.Clientset, verifyLi
 	// the link existence return false
 	return !verifyLinked
 }
+
+func setIngressNameWithRetry(ctx context.Context, namespace string, obj runtime.Object, ingressClassName *string) error {
```

Comment:
This sets an ingress **class** not an ingress name, right?

Reply:
So your suggestion is change the last parameter to `ingressClass`?

Reply:
Change the name :) And while you're at it you can also add a godoc for this func.